examples/cfssl/README.md

```diff
@@ -0,0 +1,98 @@
+# DEPRECATED
+We have replaced `cfssl` with `cert-manager` for CockroachDB certificates. Please refer to [cert-manager examples](https://github.com/utilitywarehouse/cockroachdb-manifests/examples/cert-manager) for up-to-date manifests.
```

Comment:
Is everything in this doc deprecated?

Including sections about Client, Admin UI, etc?